crosshair/libimpl/builtinslib_test.py

```diff
@@ -441,6 +441,13 @@ def f(a: str) -> int:
 
         self.assertEqual(*check_ok(f))
 
+    def test_ljust_fail(self) -> None:
+        def f(s: str) -> str:
+            """ post: _ == len(s) """
+            return s.ljust(3, "x")
+
+        self.assertEqual(*check_fail(f))
```

Comment:
So, this fails, but not in the way you intended, I suspect. `ljust` will return a string, so that won't ever equal the length of `s`.
Same for `rjust` below.

Reply:
Oh yeah definitely not what I was going for, good catch!